### ISSUE TITLE
calculating the new buildings 

### ISSUE THREAD

**Issue by chrisgraf** [boku-ilen/landscapelab#124]
after #109 is done

**Comment by kb173** [comment]
#129 also needs to be done until we can generate all

**Comment by kb173** [comment]
Unfortunately not all buildings could be converted to DSCN scenes yet because we need more disk space. However, I generated some and checked one out in Godot, and it looks good! (Screenshot made over VNC so the colors aren't perfect)

![image](https://user-images.githubusercontent.com/18697097/65032958-7aa2bd80-d944-11e9-84cd-e6d87651a800.png)

**Comment by kb173** [comment]
The last step is postponed until Thursday when we have the new SSD. All there's left to do is running the Godot conversion script which is quite fast, should only take about 10 minutes.
